server/src/logic/handler/validation/auth-validation.handler.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, Response, Router } from "express";
+import { AuthValidationMiddleware } from "./../../../logic/middleware/index";
+import { ApiError, DbError, Recipe, ValidationError } from "./../../../models/index";
+
+export class AuthValidationHandler {
+
+  constructor (private _authValidationMiddleware: AuthValidationMiddleware) {}
+
+  public checkRequest = (req: Request, res: Response, next: NextFunction): void => {
+    this._authValidationMiddleware.checkRequest(req)
+      .then(() => next())
+      .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
```

Comment:
since this seems to be always the same you could give this function a name and just use this name in the catch blocks